securedrop/sass/modules/_flash.sass

```diff
@@ -3,9 +3,9 @@
     display: flex
     flex-flow: row nowrap
     align-items: flex-start
-    border-radius: 10px
-    padding: 10px
+    border-radius: 2px
     margin: 10px
+    padding: 10px 16px
```

Comment:
This doesn't quite have the intended effect; it adds a bit more padding than needed (note the amount of whitespace around the icon compared with Nina's mockup), and also changes the styling other flash messages. Instead, I would leave this value at `10px`, and add `padding-left:16px` or similar to the `.important` specification.

Reply:
fixed in 0acdd39; was worried about internationalization so I have a rtl version as well.